services/web-server/src/main.js

```diff
@@ -149,7 +149,6 @@ const load = loader(
         const server = new ApolloServer({
           schema,
           formatError,
-          tracing: true,
```

Comment:
typescript says this option does not exist.. I tend to believe, since this was added > 7y ago

Reply:
https://github.com/apollographql/apollo-server/commit/06a9532d3fda6ac3637748dc9aa5b0697703032f

Yup, typescript is right